.github/workflows/push.yml

```diff
@@ -55,3 +55,26 @@ jobs:
 
       - name: Check Unit Tests
         run: mvn --errors test
+
+  commit-message:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Validate Tag
+        run: |
+          TAG=$(echo ${{ github.event.pull_request.title }} | sed -ne 's/\[\(.*\)\].*/\1/p')
+          echo "Found tag {$TAG}"
+          # The status code for grep will be 1 if no match, and 0 if a match.
+          # Store in variable to hide the actual grep result from the console output.
+          MATCH=$(cat .codegen/changelog_config.yml | grep "tag: \"\[$(echo $TAG)\]\"")
```

Comment:
nit: We could directly call grep on file instead of cat and grep. `-q` in case we want to suppress output of matched line. 

`````
if grep -q "tag: \"[$TAG]\"" .codegen/changelog_config.yml; then
  echo "Match found."
else
  echo "Match not found. Exiting..."
  exit 1
fi
`````